app/javascript/controllers/app/theme-switch_controller.js

```diff
@@ -1,40 +1,43 @@
 import { Controller } from "stimulus"
 
-export default class extends Controller {
-  static targets = ["switch"]
-
-  connect() {
-    const osDarkMode = window.matchMedia('(prefers-color-scheme: dark)').matches
-    const supportsLocalStorage = 'localStorage' in window
+export const setTheme = (switchTarget) => {
```

Comment:
I think it would be beneficial if we keep our JS application code consistent by continuing to use `class` objects rather then exporting a bunch of different functions/constants. What do you think?

Reply:
Thanks for the comment! I agree with that.
I exported `setTheme` separately from `class`, since we only need the code to set the colors when initially loading.
I don't think the "stimulus" at the first load is necessary.
How about split `setTheme` code to new file from this file? (and import it to this file)